mne/channels/__init__.py

```diff
@@ -7,7 +7,30 @@
                      find_layout, generate_2d_layout)
 from .montage import (read_montage, read_dig_montage, Montage, DigMontage,
                       get_builtin_montages, make_dig_montage,
-                      read_dig_egi, read_dig_captrack, read_dig_fif)
+                      read_dig_egi, read_dig_captrack, read_dig_fif,
+                      read_dig_polhemus_isotrak, read_polhemus_fastscan,
+                      compute_dev_head_t)
 from .channels import (equalize_channels, rename_channels, fix_mag_coil_types,
                        read_ch_connectivity, _get_ch_type,
                        find_ch_connectivity, make_1020_channel_selections)
+
+__all__ = [
```

Comment:
I have only ever seen `__all__` as a list of strings.

Also in general we don't use `__all__`. It might be better to impose this organization at the `python_reference.rst` level instead.